ReactWindows/ReactNative.Shared/Modules/Network/NetworkingModule.cs

```diff
@@ -355,11 +391,21 @@ public void OnDestroy()
         private void OnResponseReceived(int requestId, HttpResponseMessage response)
         {
             var headerData = new JObject();
+#if WINDOWS_UWP
             TranslateHeaders(headerData, response.Headers);
+#else
```

Comment:
encapsulate differences in a new method called getHeadersDictionary() and consolidate #ifs there